src/main/java/frc/robot/commands/Teleop/DropAlgaeIntake.java

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands.Teleop;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.Constants;
+import frc.robot.subsystems.AlgaePivot;
+
+/* You should consider using the more terse Command factories API instead https://docs.wpilib.org/en/stable/docs/software/commandbased/organizing-command-based.html#defining-commands */
+public class DropAlgaeIntake extends Command {
+  /** Creates a new PivotAlgae. */
+  private AlgaePivot intakePivot;
+  private boolean done = false;
+  public DropAlgaeIntake(AlgaePivot intakePivot) {
+    this.intakePivot = intakePivot;
+    addRequirements(intakePivot);
+    // Use addRequirements() here to declare subsystem dependencies.
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    intakePivot.setPosition(Constants.AlgaePivot.DROPPED);
+    done = true;
```

Comment:
don't do this. You need to be checking for position in isFinished to know when to end the command